packages/shared-common/src/index.ts

```diff
@@ -78,7 +78,7 @@ export function getMatchedPositions(code: string, matched: string[], hasVariantG
 
   // attributify values
   attributify.forEach(([, name, value]) => {
-    const regex = new RegExp(`(${e(name)}=)(['"])[^\\2]*?${e(value)}[^\\2]*?\\2`, 'g')
+    const regex = new RegExp(`(${name}=)(['"])[^\\2]*?${value}[^\\2]*?\\2`, 'g')
```

Comment:
Shouldn't `e` at least be replaced with `escapeRegEx`?

Reply:
I looked at this function, but I'm sorry, I didn't figure out what it means. Can you explain it in detail?

Reply:
> I looked at this function, but I'm sorry, I didn't figure out what it means. Can you explain it in detail?

https://developer.mozilla.org/en-US/docs/Web/API/CSS/escape

Reply:
I'm sorry the function name was `escapeRegExp`. (see [search](https://github.com/unocss/unocss/search?q=escapeRegExp))
https://github.com/unocss/unocss/blob/d337ce401788a898208e3ebdb94b2e6c3c0ab01b/packages/core/src/utils/escape.ts#L1

This is usually done if you are using string from variable while constructing regex, similar to escaping string variable with `e()` or the like before using it in html.

Reply:
@chu121su12 Thanks, you are right, I have replaced `e()` with `escapeRegExp()` and added tests.